### DISCUSSION TITLE
MCT 9.0.3 not on nuget.org (resolved)

### DISCUSSION THREAD

**Discussion by stephenquan** [CommunityToolkit/Maui#2130] [Q&A] [answered]
MCT 9.0.3 just got released.
However, nuget.org only sees MCT 9.0.2.
I tried adding either `<MauiVersion>8.0.71</MauiVersion>` or `<MauiVersion>8.0.80</MauiVersion>` to my project but it didn't work.

https://www.nuget.org/packages/CommunityToolkit.Maui shows:

<img src="https://github.com/user-attachments/assets/62a4fac4-3438-4ead-97d0-c33f50e4b3ba" height="200" />

Visual Studio 2022 Manage NuGet packages shows:

<img src="https://github.com/user-attachments/assets/cba24095-efbf-40e6-a59b-69a621745b18" height="200" />

References:
 - https://github.com/CommunityToolkit/Maui/releases/tag/9.0.3
 - https://www.linkedin.com/posts/jfversluis_releases-communitytoolkitmaui-activity-7230208399047602177-6W2T?utm_source=share&utm_medium=member_desktop

**Comment by TheCodeTraveler** [comment]
@jfversluis Any inkling to what's broken in our pipeline?

**Reply by jfversluis** [reply, marked as answer]
Unfortunately not! Our release says it uploaded successfully, so I'm trying to find someone who can tell me why it isn't showing up 😄

**Comment by CliffAgius** [comment]
Closed as Answered.
